src/chrome/content/rules/Raiffeisen.xml

```diff
@@ -10,6 +6,7 @@ Fetch error: http://tb.raiffeisendirect.ch/ => https://tb.raiffeisendirect.ch/:
         <securecookie host="^(?:.*\.)?raiffeisen\.ch$" name=".*" />
         <securecookie host="^(?:.*\.)?raiffeisendirect\.ch$" name=".*" />
 
-        <rule from="^http://(?:www\.)?raiffeisen\.ch/" to="https://www.raiffeisen.ch/"/>
```

Comment:
The previous rule would send `raiffeisen.ch` to `www.raiffeisen.ch`, but the new one won't. Is that intentional?